src/destructors.md

```diff
@@ -3,10 +3,11 @@ r[destructors]
 
 r[destructors.intro]
 When an [initialized]&#32;[variable] or [temporary] goes out of
-[scope](#drop-scopes), its *destructor* is run, or it is *dropped*. [Assignment]
-also runs the destructor of its left-hand operand, if it's initialized. If a
-variable has been partially initialized, only its initialized fields are
-dropped.
+[scope](#drop-scopes), it is *dropped*, which means its *destructor* is run.
+(The two terms are used interchangeably here.)
```

Comment:
*dropped* and *destructor* aren't the two terms used interchangeably.  (We use the emphasis to define terms, so what's under emphasis matters here.)  While extending the emphasis on the latter to include "is run" would be one option, that reads strangely to me still, so maybe there are other options here.

It'd seem better to me if we could find a factoring that avoided this parenthetical.  This is just an introduction.  Maybe something like reversing the terms back to the original order and saying something like "When a value or temporary goes out of scope and its destructor is run, we say that it has been *dropped*."

Reply:
(I'm happy to make the revision here if something along these lines sounds OK to you.)